packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -21,12 +23,15 @@ const nonSampledTraceParent = `00-10-10-00`
 
 type Options = {
   traceMiddleware: boolean
+  ignoreLayersTypes: PrismaLayerType[]
```

Comment:
Wouldn't it be more flexible to use hooks instead of a list of span names to filter? 
Beyond filtering spans, we could consider adding or transforming attributes (on engine or client spans). 
This is what is done in the HTTP instrumentation: https://www.npmjs.com/package/@opentelemetry/instrumentation-http (`applyCustomAttributesOnSpan`, `requestHook`, ...) and it's quite an interesting approach.